### ISSUE TITLE
Switching Participant to Local and vice versa not working

### ISSUE THREAD

**Issue by codemeall** [blackuy/react-native-twilio-video-webrtc#219]
Switching Participant View to Local and at the same time Switching the thumbnail respectively doesn't work in android, IOS it works fine, need help with this, is there any thing additional I have to do in Android?

**Comment by codemeall** [comment]
the local and participant views are changing fine but when Local Video View is switch to primary then it overlaps the participant thumbnail, need help to change the default zindex or the default overlap of Local Video

**Comment by slycoder** [comment]
Hi, just wanted to get some clarification.  Are you saying that the views do not respect the `zIndex` style property?

**Comment by codemeall** [comment]
Yes Exactly, when I make LocalVideo as primary it overlaps the Peer's Video thumbnail, and I changed the `applyZorder` to false in twilioVideoPreview file of android and it worked fine then, need to make `applyZorder` dynamic for android.

**Comment by slycoder** [comment]
Oh yeah, that makes sense!  I'm a bit busy at the moment; it sounds like you know where the issue is so if you wanna send a PR I'd be happy to merge.

**Comment by codemeall** [comment]
Sure, I'll make a PR.

**Comment by DanielMenke** [comment]
@codemeall Hey, did you already created the PR?

**Comment by RonicleyNuvidio** [comment]
I have the same issue. Does anyone have any solutions?

**Comment by DanielMenke** [comment]
@RonicleyNuvidio 
More of a workaround, but this seems to work:

change your `node_modules/react-native-twilio-video-webrtc/android/src/main/java/com/twiliorn/library`
to:
```
 package com.twiliorn.library;
 import com.facebook.react.bridge.WritableMap;
 import com.facebook.react.uimanager.ThemedReactContext;

 public class TwilioVideoPreview extends RNVideoViewGroup {

     private static final String TAG = "TwilioVideoPreview";

     public TwilioVideoPreview(ThemedReactContext themedReactContext) {
         super(themedReactContext);
         CustomTwilioVideoView.registerThumbnailVideoView(this.getSurfaceViewRenderer());
         this.getSurfaceViewRenderer()
         .applyZOrder(false);  // <==== THIS IS IMPORTANT!!
     }
 }
```

And build again

**Comment by RonicleyNuvidio** [comment]
@DanielMenke Just changing to false works on both ios and android?

**Comment by DanielMenke** [comment]
@RonicleyNuvidio No it just fixes the Android build. Had no problems with iOS, luckily.

**Comment by RonicleyNuvidio** [comment]
Tks @DanielMenke

**Comment by RonicleyNuvidio** [comment]
@DanielMenke Do you know if someone makes a pull request to fix this problem?

**Comment by DanielMenke** [comment]
@RonicleyNuvidio I don't think that this ist a proper solution. Its more of a workaround. @slycoder what do you say?

**Comment by slycoder** [comment]
Hey all, please give #375 a try and see if that fixes the issue for you.  That PR adds a new prop to configure the z indexes on android.
